### PR TITLE
Release eds tokens 1.0.0 beta 1

### PR DESCRIPTION
**Pull request by oddvernes**
This pr must not be merged, it is only for inspection

### CODE REVIEW

File: packages/eds-tokens-build/src/index.ts [hunk 1]
```diff
@@ -329,7 +332,7 @@ const _extend = ({
           },
         ],
       },
-      ts: {
+      /*       ts: {
```
Comment (torleifhalseth) [inline packages/eds-tokens-build/src/index.ts:335]: Why are we excluding the ts and JSON transforms?

File: packages/eds-tokens/package.json [hunk 2]
```diff
@@ -55,7 +42,7 @@
     "elements.css"
   ],
   "scripts": {
-    "build": "rollup -c --bundleConfigAsCjs && pnpm run types && pnpm --filter @equinor/eds-tokens-sync run build && pnpm --filter @equinor/eds-tokens-build run build",
+    "build": "rollup -c --bundleConfigAsCjs && pnpm run types",
```
Comment (torleifhalseth) [inline packages/eds-tokens/package.json:45]: I guess we can assume that the token-sync and token-build are pre-build, but I think it makes sense to add `tokens-in-code:build` here. Then the build of all the things are isolated in the package, and we don't have to specify both the old and new build in the root package.json.

File: package.json [hunk 3]
```diff
@@ -8,12 +8,14 @@
     "lint:lab": "pnpm run lint ./packages/eds-lab-react",
     "lint:utils": "pnpm run lint ./packages/eds-utils",
     "init": "pnpm install --force && pnpm run build",
-    "build": "pnpm run build:icons && pnpm run build:tokens && pnpm run build:utils && pnpm run build:core-react && pnpm run build:data-grid-react && pnpm run build:lab",
+    "build": "pnpm run build:icons && pnpm run build:tokens && pnpm run build:utils && pnpm run build:core-react && pnpm run build:data-grid-react && pnpm run build:lab && pnpm run build:token-build && pnpm run build:token-sync",
```
[the remark this answers is not in the document]
Comment (oddvernes) [reply package.json:11]: It should not be run as part of build, it should be manually ran by a human and comitted same as we do with the old tokens

File: packages/eds-tokens-build/src/index.ts (hunk 1 above)
Comment (oddvernes) [reply packages/eds-tokens-build/src/index.ts:335]: they were misnamed (-verbose) and we didn't have time to fix it

File: packages/eds-tokens/package.json (hunk 2 above)
Comment (oddvernes) [reply packages/eds-tokens/package.json:45]: No they absoluetly should be seperate

Comment (oddvernes) [reply packages/eds-tokens/package.json:45]: `tokens-in-code:build` should be considered as part of the transform step. It is only ran when we do `transform:tokens`. The reason for this is that `lightningcss` is an external dependency that will get updated over time which could affect the output. Changes to output must always be verified to make sure nothing unexpectely changes. Unlike 1.0 tokens, 2.0 tokens does not have a build step, only sync and transform

**Pull request closed without merging**